src/api/forms/service.js

```diff
@@ -96,7 +96,7 @@ function formTitleToId(formTitle) {
 
 /**
  * Retrieves the empty form configuration
- * @returns {Promise<object>} - the empty form configuration
+ * @returns {Promise<import('@defra/forms-model').FormDefinition>} - the empty form configuration
  * @throws {InvalidFormDefinitionError} - if the base form definition is invalid
  */
 async function retrieveEmptyForm() {
```

Comment:
this will need an allow unsafe return

Reply:
I've made them all warnings in https://github.com/DEFRA/forms-manager/pull/10/commits/42fb195b35c371b9d4212f340d0610c0bfaf9e69

We can slowly fix them as necessary, rather than hide them away to be forgotten